index.html

```diff
@@ -119,7 +122,10 @@ <h2 class="year">2015</h2>
                         <li><a href="#"><img src="./assets/icons/jslogo.png" alt="jslogo"></a></li>
                     </ul>
                     <div class="button-container">
-                        <button class="button-see-project" type="button">
+                        <button class="button-see-project mobile-button-pp" type="button" id="1">
+                            See project
+                        </button>
+                        <button class="button-see-project desktop-button-pp" type="button" id="5">
```

Comment:
There should be no project that is manually created using HTML. The projects section should be created dynamically with the information from the projects array as per project requirements.

![image](https://user-images.githubusercontent.com/38088818/174059434-460fc8f0-8627-4c2d-9d52-11a5133905ad.png)

-  This means that all the HTML of the `works` section is created with JavaScript when the page loads. **This requires changes in existing code - already created HTML code for the projects list should be removed, and instead it should be added via JavaScript.**
- As a result, you should have both the projects and the popup being rendered dynamically with the projects array you created.